src/instructlab/model/chat.py

```diff
@@ -55,6 +67,211 @@
 PROMPT_PREFIX = ">>> "
 
 
+@click.command()
+@click.argument(
+    "question",
+    nargs=-1,
+    type=click.UNPROCESSED,
+)
+@click.option(
+    "-m",
+    "--model",
+    default=helpers.DEFAULT_MODEL,
+    show_default=True,
+    help="Model to use",
+)
+@click.option(
+    "-c",
+    "--context",
+    default="default",
+    show_default=True,
+    help="Name of system context in config file.",
+)
+@click.option(
+    "-s",
+    "--session",
+    type=str,
+    default=None,
+    help="Filepath of a dialog session file.",
+)
+@click.option(
+    "-qq",
+    "--quick-question",
+    is_flag=True,
+    help="Exit after answering question.",
+)
+@click.option(
+    "-gm",
+    "--greedy-mode",
+    is_flag=True,
+    help="Use model greedy decoding. Useful for debugging and reproducing errors.",
+)
+@click.option(
+    "--max-tokens",
+    type=click.INT,
+    help="Set a maximum number of tokens to request from the model endpoint.",
+)
+@click.option(
+    "--endpoint-url",
+    type=click.STRING,
+    help="Custom URL endpoint for OpenAI-compatible API. Defaults to the `ilab serve` endpoint.",
+)
+@click.option(
+    "--api-key",
+    type=click.STRING,
+    default=DEFAULT_API_KEY,  # Note: do not expose default API key
```

Comment:
Comment needed?

Reply:
Also, should this be `helpers.DEFAULT_API_KEY`?

Reply:
comment was there, and I imported the vars directly from `config` so no need for the `.DEFAULT...`